infrastructure/main.tf

```diff
@@ -76,7 +77,20 @@ module "key-vault" {
   product_group_object_id = "38f9dea6-e861-4a50-9e73-21e64f563537"
 }
 
+# used in output
+data "azurerm_key_vault" "vault" {
+  name = "${local.vault_name}"
+  resource_group_name = "${local.vault_rg}"
+}
+
 data "azurerm_key_vault_secret" "test_service_notify_api_key" {
   name = "test-service-notify-api-key"
   vault_uri = "${module.key-vault.key_vault_uri}"
 }
+
+# used by jenkins to set env var for smoke test
```

Comment:
It doesn't even have to be tests - we can store this secret and the 'official' connection string to the topic.